ecs-agent/ipcompatibility/ipcompatibility.go

```diff
@@ -34,6 +34,11 @@ func NewIPv6OnlyCompatibility() IPCompatibility {
 	return NewIPCompatibility(false, true)
 }
 
+// NewDualStackCompatibility returns a dual-stack IPCompatibility value.
+func NewDualStackCompatibility() IPCompatibility {
+	return NewIPCompatibility(true, true)
+}
+
```

Comment:
Do we need a test using this specific new method?

Reply:
Yep. Added.